src/server/pegasus_read_service.h

```diff
@@ -79,6 +81,7 @@ class pegasus_read_service : public dsn::replication::replication_app_base,
         register_rpc_handler_with_rpc_holder(dsn::apps::RPC_RRDB_RRDB_SCAN, "scan", on_scan);
         register_async_rpc_handler(
             dsn::apps::RPC_RRDB_RRDB_CLEAR_SCANNER, "clear_scanner", on_clear_scanner);
+        register_rpc_handler_with_rpc_holder(RPC_DETECT_HOTKEY, "detect_hotkey", on_detect_hotkey);
```

Comment:
I still don't recommend that the `detect_hotkey` rpc handler is registered in `pegasus_read_service`.